src/components/CoinTable/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Coin } from '../../types/types';
+import styles from './CoinTable.module.scss';
+
+const {
+  coins,
+  coins__table,
+  coins__name,
+  coins__symbol,
+  coins__price,
+  coins__volume,
+  coins__rank,
+} = styles;
+
+type CoinTableProps = {
+  coinList: Coin[];
+};
```

Comment:
I suggest creating a separate file within this folder and moving the types into this file